next.config.js

```diff
@@ -18,8 +19,11 @@ const withMDX = require('@next/mdx')({
     ],
     rehypePlugins: [],
   },
-})
-module.exports = withMDX({
+});
+
+const withTM = require("next-transpile-modules")(["next-i18next-static-site"]);
```

Comment:
Why you need the `next-transpile-modules`?

Reply:
I was following the guide for next-18next and got a demo project for that. There, I saw it being used with ```next-18next-static-site```